js/index.js

```diff
@@ -0,0 +1,97 @@
+
+var calcChoice = prompt('Would you like to use the basic or the advance calculator? for Advance press 1 for Basic press 2 and press 3 to calculate your BMI');
```

Comment:
These prompts are quite awkward and difficult to read. Try to keep consideration of user experience in your head in the future.